driver-adapters-wasm/pg-node-basic-wasm-edge-import/index.mjs

```diff
@@ -0,0 +1,160 @@
+// @ts-check
+import { Prisma, PrismaClient } from '@prisma/client'
+import { PrismaPg } from '@prisma/adapter-pg'
+import pkg from 'pg'
+const { Pool } = pkg
```

Comment:
Can this be done cleaner?

Reply:
I think that is how I started, and then worked through the error messages until I ended up where I did :D I wonder what went wrong...